platform_api/orchestrator/kube_orchestrator_preemption.py

```diff
@@ -132,60 +154,18 @@ def get_pod_node_name(self, pod_name: str) -> str | None:
         """
         return self._pod_node_names.get(pod_name)
 
-    def _get_node_allocatable_resources(self, node_name: str) -> NodeResources:
-        node = self._nodes.get(node_name)
-        return node.allocatable_resources if node else NodeResources()
-
-
-class IdlePodsHandler(PodEventHandler):
```

Comment:
Don't need to track idle pods, assume that these resources are free for scheduling.